package.json

```diff
@@ -32,6 +32,8 @@
     "release:version-bump": "lerna version --exact",
     "release:start-ci-publish": "node -p '`Publish (dist-tag:${process.env.APOLLO_DIST_TAG || \"latest\"})`' | git tag -F - \"publish/$(date -u '+%Y%m%d%H%M%S')\" && git push origin \"$(git describe --match='publish/*' --tags --exact-match HEAD)\"",
     "package-extension": "npm run clean && npm ci && lerna run package-extension --stream --scope=vscode-apollo",
+    "package-extension:no-clean": "lerna run package-extension --stream --scope=vscode-apollo",
```

Comment:
On CI, this is a fresh install, so we don't need to clean and reinstall. I want to leave it for local builds though just as a safety measure